jsconfig.json

```diff
@@ -1,11 +1,5 @@
 {
-    "compilerOptions": {
-      "baseUrl": "./src",
-      "paths": {
-        "@/components/*": ["components/*"],
-        "@/contexts/*": ["contexts/*"],
-        "@/common/*": ["common/*"],
-        "@/serviceFacades/*":["@/serviceFacades/*"]
```

Comment:
We aren't using these anywhere and since they break storybook it seems like they're better removed.